devel/python/python/ert_gui/models/connectors/run/multiple_data_assimilation.py

```diff
@@ -50,8 +50,10 @@ def runSimulations(self):
         iteration_count = len(weights)
 
         print("Running MDA ES for %s  iterations\t%s" % (iteration_count, ", ".join(str(weight) for weight in weights)))
-        weights = self.normalizeWeights(weights)
 
+        self.setAnalysisModule()
```

Comment:
I do not understand this:

1. An analysis module has already been assigned - either the default module, or something from the configuration file.
2. I would assume the `setAnalysisModule()` method would require an argument?